ras_party/controllers/error_decorator.py

```diff
@@ -2,9 +2,9 @@
 
 import structlog
 from flask import current_app
+from ras_common_utils.ras_error.ras_error import RasError
```

Comment:
Same as other comment - a pickie point i know and only change if you have time and can be arsed.